packages/google_sign_in/google_sign_in/android/src/test/java/io/flutter/plugins/googlesignin/GoogleSignInTest.java

```diff
@@ -0,0 +1,16 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.googlesignin;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class GoogleSignInTest {
+  @Test
+  public void aTest() {
+    assertEquals(2, 2);
```

Comment:
is there something from the plugin implementation that we could test?

Reply:
I could, but I think this would require me to change the code of each plugin to add a meaningful test. The Java code in most of these plugins weren't designed to be tested and adding a test would require making a method/field public and mocking stuff. I'll add some arbitrary tests

Reply:
@stuartmorgan any thoughts?

Reply:
The primary goal is to have the structure in place so that we can point PR authors to a file to add tests to, rather than saying "you need to add a test file in the right place, then update the build, etc."

A meaningful test is obviously better than not since it provides more of an example to follow, but I'm fine with just having a placeholder in the interests of getting the basic harness infrastructure in place everywhere. If we do placeholders we should:
- Clearly comment that it's a placeholder until some real tests are in place (I know it should be obvious, but a first-time contributor could be confused).
- Ideally have the placeholder do *something* with the plugin (e.g., in the corresponding iOS changes Jenn had the placeholder test just instantiate the plugin; could we do something similar?). That guarantees that we don't have, say, linking or import issues that someone trying to use it for the first time would immediately hit.